README.md

```diff
@@ -95,12 +95,75 @@ our [Contributing](https://github.com/cyberark/conjur-api-java/blob/master/CONTR
     or
     ```sh-session
     $ mvn install:install-file -Dfile=/path/to/api/repo/target/conjur-api-$VERSION-with-dependencies.jar \
-        -DgroupId=net.conjur.api \
+        -DgroupId=com.cyberark.conjur.api \
         -DartifactId=conjur-api \
         -Dversion=$VERSION \
         -Dpackaging=jar
     ```
 
+### Using Maven Releases
+To make use of tagged releases published to Maven, verify that you have the dependency 
+added to your `pom.xml`
+
+1. Add the following snippet to `pom.xml`
+```xml
+<dependency>
+  <groupId>com.cyberark.conjur.api</groupId>
+  <artifactId>conjur-java-api</artifactId>
+  <version>x.x.x-SNAPSHOT</version>
```

Comment:
I'm guessing that `SNAPSHOT` part in here is not right since these will be regular releases pulled when used this way.

Reply:
Correct - fixed!